packages/@adobe/spectrum-css-temp/components/dialog/index.css

```diff
@@ -301,6 +301,28 @@ governing permissions and limitations under the License.
   }
 }
 
+/* additional a11y support by moveing scrolling from content to dialog */
+@media screen and (max-height: 400px) {
+  .spectrum-Dialog .spectrum-Dialog-grid {
+    border-top-left-radius: var(--spectrum-dialog-border-radius);
+    border-top-right-radius: var(--spectrum-dialog-border-radius);
+    overflow-y: auto;
+  }
+
+  .spectrum-Dialog-content {
+    overflow-y: visible;
+    height: min-content;
```

Comment:
why not `auto`?

Reply:
o, i see, it has a row definition of 1fr, this is fine, though if we wanted to be a bit more correct, we could change the row definition to `auto` and height to `auto`, but this pretty much accomplishes that at the moment